test/schemas/v0-0-1/formatted-parts.schema.json

```
@@ -0,0 +1,206 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/unicode-org/message-format-wg/main/test/schemas/v0-0-1/formatted-parts.schema.json",
+  "title": "Formatted parts",
+  "description": "The expected result of formatting the message to parts.",
+  "type": "array",
+  "items": {
+    "oneOf": [
+      {
+        "$ref": "#/$defs/messageLiteralPart"
+      },
+      {
+        "$ref": "#/$defs/messageMarkupPart"
+      },
+      {
+        "$ref": "#/$defs/messageExpressionPart"
+      }
+    ]
+  },
+  "$defs": {
+    "messageLiteralPart": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "type",
+        "value"
+      ],
+      "properties": {
+        "type": {
+          "const": "literal"
+        },
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "messageMarkupPart": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "type",
+        "kind",
+        "name"
+      ],
+      "properties": {
+        "type": {
+          "const": "markup"
+        },
+        "kind": {
+          "enum": [
+            "open",
+            "standalone",
+            "close"
+          ]
+        },
+        "source": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "options": {
+          "type": "object"
+        }
+      }
+    },
+    "messageExpressionPart": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "datetime",
+            "fallback",
+            "number",
+            "string",
+            "unknown"
+          ]
+        }
```

This is still too strict, and unnecessary. We do not need to make this parts schema require extension every time a new function is added. All of the subtypes of this should be dropped, leaving `messageExpressionPart` to be as freeform as it really is.

```suggestion
        "type": "string"
```

---

Sounds good. Updated.